addons/packages/pinniped/0.12.1/bundle/config/overlay/update-strategy-overlay.yaml

```diff
@@ -48,3 +48,19 @@ spec:
   updateStrategy:
     type: #@ data.values.daemonset.updateStrategy
   #@ end
+
+#@overlay/match expects="0+",by=overlay.subset({"kind":"Job"})
```

Comment:
question: do you think `expects="1"` would be more accurate here? since we always deploy a single `Job`?

Reply:
Yeah I can make that swap quick!